docs/release_process.md

```diff
@@ -7,9 +7,59 @@ This document describes the bpftrace release process.
 We choose to follow semantic versioning. Note that this doesn't matter much for
 major version < 1 but will matter a lot for >= 1.0.0 releases.
 
-See https://semver.org/ .
+See https://semver.org/.
 
-## Branching model
+## Release cadence
+
+bpftrace is released twice a year. Since our biggest dependency, which also
+tends to break things, is LLVM, we align with the [LLVM release
+schedule](https://llvm.org/docs/HowToReleaseLLVM.html). In particular, a
+bpftrace release should happen **two weeks after a major LLVM release**.
+
+In addition, four weeks before the bpftrace release, we create a stabilized
+release branch, which will only receive bug fixes from that moment.
```

Comment:
nit
```suggestion
release branch, which will only receive bug fixes affecting the release branch.
```

Reply:
Hm, that sounds a bit weird to me, too much repetition of "release branch". How about
```suggestion
release branch, which will only receive bug fixes affecting the release itself.
```